test/Utils.test.js

```diff
@@ -0,0 +1,69 @@
+import {
+  getSecondsFromNow,
+  getEpochSecondsFromWSDOT,
+  getCurrentEpochSeconds,
+  getProgress,
+  calculateDistance,
+} from '../src/Utils.js';
+// import jest from 'jest-mock';
```

Comment:
Remove this line?

Reply:
will do. missed this when cleaning up lint errors.